ci/infra/azure/master-instance.tf

```diff
@@ -70,3 +77,16 @@ resource "azurerm_virtual_machine_extension" "master" {
     }
 SETTINGS
 }
+
+locals {
+  master_principal_ids = var.cpi_enable ? azurerm_linux_virtual_machine.master.*.identity.0.principal_id : []
+}
+
+resource "azurerm_role_assignment" "master" {
```

Comment:
What role are you assigning to the masters here ?

It should be an option to manage this with terraform, as it requires the Operator to have full control over the subscription, which is not an ideal security model (at least we wanted not QA engineers to have 'Owner' role over SUSE Engineering subscription as a concrete example)
Instead documentation should explain how the owner could provide the required configuration.